pkgs/by-name/mo/morewaita-icon-theme/package.nix

```
@@ -7,13 +7,13 @@
 }:
 stdenvNoCC.mkDerivation rec {
   pname = "morewaita-icon-theme";
-  version = "47.1";
+  version = "47.2";
 
   src = fetchFromGitHub {
     owner = "somepaulo";
     repo = "MoreWaita";
     rev = "${version}";
```

```suggestion
    rev = "refs/tags/v${version}";
```
Seems that upstream has started prefixing their tags with `v`, so we need to reflect that here in order to get the right URL